python/lsst/jointcal/jointcal.py

```diff
@@ -208,7 +208,8 @@ def run(self, dataRefs, profile_jointcal=False):
         if len(dataRefs) == 0:
             raise ValueError('Need a list of data references!')
 
-        jointcalControl = jointcalLib.JointcalControl(self.sourceSelector.config.sourceFluxType)
+        sourceFluxField = "slot_%sFlux" % self.sourceSelector.config.sourceFluxType
```

Comment:
I suggest `"..." % (val,)` instead of `"..." % val`. Admittedly it is uglier, but it is also safer.